torch/_refs/__init__.py

```diff
@@ -3551,6 +3552,22 @@ def t(a: TensorLikeType):
     return torch.transpose(a, 0, 0 if a.ndim < 2 else 1)
 
 
+# CompositeImplicitAutograd, but Python name doesn't match the ATen name
+@register_decomposition(torch.ops.aten.numpy_T)
```

Comment:
Why do you need a decomposition if the op is CompositeImplicit?

Reply:
`test_refs_are_in_decomp_table` fails otherwise. Likely due to a name mismatch between Python and ATen. I could add `T` to `not_in_decomp_table` instead, should I?